service/db_access.py

```diff
@@ -0,0 +1,13 @@
+from sqlalchemy import false
+from service.models import TitleRegisterData
+
+
+# TODO: write integration tests
+def get_title_register(title_ref):
+    if title_ref:
+        # Will retrieve the first matching title that is not marked as deleted
+        result = TitleRegisterData.query.filter(TitleRegisterData.title_number == title_ref,
+                                                TitleRegisterData.is_deleted == false()).first()
```

Comment:
I only moved this code from the server module. But we care because titles marked as deleted should be treated as if they didn't exist - they exist in the db only so that elasticsearch updater could find out they've been deleted.